README.rst

```diff
@@ -87,13 +89,21 @@ Table of contents
 
 - `License`_
 
+====================
+Deprecation warning
+====================
+
+    You are reading the documentation of V13. This version is currently in "Bot API compatibility mode", so will only get updates to include new Bot API features.
+    
+    If you can, you should switch your code to V20.
+
```

Comment:
can we move this to the doc-fixes-v13 pr?

Reply:
I can rollback the changes sure